assets/scss/components/foundation/dropdown/_dropdown.scss

```diff
@@ -10,6 +10,8 @@
 }
 
 .dropdown-menu {
+    z-index: 1;
```

Comment:
Could we use zIndex tool map from _z-index.scss file or do we need exactly 1 and 2 values for the fix?

Reply:
We can use $zIndex-low and $zIndex-high from zIndex tool but we do not need so big values. Should we add more variables to the tool? z-index value of .dropdown-menu-button must be higher then .dropdown-menu

Reply:
I believe, if we can use current variables and this won't affect any new issue, we should do it)

Reply:
done